### PR TITLE
Create activities

### PR DESCRIPTION
**Pull request by HUSTToanLH**

### CODE REVIEW

File: app/controllers/static_pages_controller.rb [hunk 1]
```diff
@@ -1,4 +1,12 @@
 class StaticPagesController < ApplicationController
   def home
+    if logged_in?
+      @categories = Category.all
+      @words = Word.send("by_learned", current_user.id, @categories.map(&:id)).
+      paginate page: params[:page]
```
Comment (hellovietnam93) [inline app/controllers/static_pages_controller.rb]: ident

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -6,6 +6,14 @@ def index
   end
 
   def show
+    if logged_in?
+      @categories = Category.all
+      @words = Word.send("by_learned", @user.id, @categories.map(&:id)).
+      paginate page: params[:page]
```
Comment (hellovietnam93) [inline app/controllers/users_controller.rb]: ident

File: app/views/activities/_activity.html.erb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<li class="list-group-item">
+  <h4>
+    <% unless @user.avatar.blank? %>
```
Comment (hellovietnam93) [inline app/views/activities/_activity.html.erb]: reverse it, don't use unless if don't need

File: app/views/layouts/_header.html.erb [hunk 4]
```diff
@@ -15,9 +15,34 @@
...
+            <li><%= link_to t("views.layouts.users"), users_path %></li>
+          <% end %>
+          <li class = "dropdown">
+            <a href = "#" class = "dropdown-toggle" data-toggle = "dropdown">
+              <%= t "user.account" %>
+              <b class = "caret"></b>
+            </a>
+            <ul class = "dropdown-menu">
+              <li>
+                <%= link_to t("user.profile"), user_path(current_user) %>
```
Comment (hellovietnam93) [inline app/views/layouts/_header.html.erb]: <%= link_to t("user.profile"), current_user %>

File: app/views/activities/_activity.html.erb [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+      <%= image_tag @user.avatar, size: Settings.avatar.size_img_tag %>
+    <% else %>
+      <%= image_tag Settings.avatar.default, size: Settings.avatar.size_img_tag %>
+    <% end %>
+    <span class="text-info"><%= @user.name %></span>
+    <%= activity.action %> ( <%= activity.created_at.to_date %> )
```
Comment (tqtrung09) [inline app/views/activities/_activity.html.erb]: Convention 
(<%= %>)

**Pull request merged**